### ISSUE TITLE
Improving WordPress support?

### ISSUE THREAD

**Issue by graham73may** [ddev/ddev-browsersync#51]
Thanks for the add on, I've been replacing Livereload with this and it's been great. 

I work on Drupal and WordPress websites, the Drupal projects are quite URL agnostic but the WordPress websites are constantly trying to redirect me to a URL without the :3000 port number. 

I'm not sure what functionality is available to add ons, would modifying the `wp-config-ddev.php` file be an option?

My current work around is to add the below code snippet to the auto generated `wp-config-ddev.php` file. 
* Commenting out the old `WP_HOME` and `WP_SITEURL` lines. 
* Adding some code that allows any domain. 

```
 /** WP_HOME URL */
  //  defined('WP_HOME') || define('WP_HOME', 'https://wbg.org.uk.ddev.site');

  /** WP_SITEURL location */
  //  defined('WP_SITEURL') || define('WP_SITEURL', WP_HOME . '/');

  /** Allow any domain/port. (For browsersync) */
  $domain = sprintf('%s://%s', $_SERVER['SERVER_PORT'] == 80 ? 'http' : 'https', $_SERVER['SERVER_NAME']);

  define('WP_SITEURL', $domain);
  define('WP_HOME', $domain);
```

Maybe it would be better to only allow the project URL(s) in their original form and with `:3000` so it's not too loose? 

Thanks,

**Comment by tyler36** [comment]
Hi @graham73may 

Thanks for the report. I don't use WP so I'm not sure the best solution. It would be nice if we did have something though.

**Comment by graham73may** [comment]
Is it possible for an addon to modify the ddev-generated `wp-config-ddev.php` file? 

Failing that, is it possible for the addon to detect the DDEV project type === "wordpress" and then echo some tips into the console?

**Comment by rfay** [comment]
It is in fact possible, see how ddev/ddev-redis does it for Drupal config. And absolutely possible to output help when detecting project type as well.

**Comment by tyler36** [comment]
@graham73may 
Any progress on the PR? Did you hit a problem? There's usually some friendly folks over on Discord if you need help. 😄

**Comment by graham73may** [comment]
Hey @tyler36, sorry project work has been taking priority. With the code snippet above working it's been making me lazy. Right now is actually not a bad time, taking a look at what ddev-redis is doing. Will report back shortly.

**Comment by tyler36** [comment]
Fixed in #57
